src/ai/backend/client/cli/ssh.py

```diff
@@ -12,14 +12,20 @@
 from .pretty import print_fail, print_info
 
 
+def find_backendai_executable():
+    if __debug__ and os.path.isfile("./backend.ai"):
+        return "./backend.ai"
+    return "backend.ai"
```

Comment:
I think it would be better to use:
```suggestion
    return [sys.prefix, "-m", "ai.backend.cli"]
```